probe-rs/src/probe/stlink/mod.rs

```diff
@@ -852,9 +852,13 @@ impl<D: StLinkUsb> STLink<D> {
                 "8-Bit reads are limited to 64 bytes on ST-Link v2"
             );
         } else {
+            // This 128 byte limitation was empirically derived by @disasm @diondokter and @Yatekii
+            // on various STM32 chips and different ST-Linkv3 versions (J4, J5, J6, J7).
+            // It works until 255. 256 and above fail. Apparently it *should* work with up to
```

Comment:
I checked this 255-byte limitation only on V3J5.

Reply:
I checked on J7 So imma change this.